### ISSUE TITLE
Not receiving events

### ISSUE THREAD

**Issue by karimApps142** [kingstinct/react-native-device-activity#10]
I am facing an issue where I am not receiving any events despite having granted the necessary permissions and set the threshold to 1 minute. I have also selected all apps for monitoring.

**Comment by karimApps142** [comment]
"I received the following log message in the Xcode console window:

apache
Copy
DeviceActivity.AppLoggedTimeDaily.17
2023-08-18 11:27:27.555018+0500 Timeout[2755:926609] [react-native-device-activity] ✅ Succeeded with Starting Monitor Activity: DeviceActivity.AppLoggedTimeDaily.18
2023-08-18 11:27:27.655758+0500 Timeout[2755:926609] [react-native-device-activity] ❌ Failed with Starting Monitor Activity: This application is monitoring excessive activities
Please note that the log message was not generated from the React Native bundler, but from the Xcode console window while running the project. I copy the code from the example file .

**Comment by robertherber** [comment]
Yeah, I think the example isn't quite working right now. Don't have time to dig further into it right now.

The "excessive activities" error is because the example tries to register 24 activities (one per hour) - and apparently there is a limit on 20. Not sure if this is what causes the events not to trigger for you, or if the first 20 activities still work as expected.

One thing that another developer noted is that you might have to [add this to your app.json](https://docs.expo.dev/build-reference/app-extensions/). If that solves it for you we should add it to the config plugin.
